src/index.ts

```diff
@@ -4,7 +4,7 @@ export { Context } from './context'
 export * as Markup from './markup'
 export { Middleware } from './types'
 export { Router } from './router'
-export { session } from './session'
+export * from './session'
```

Comment:
Same as with #1194, I don't like the idea of overwhelming new users by exporting everything directly from entry point. I'd rather have `session`, `interface session.Storage` or `interface session.SessionStorage`, and `class session.MemorySessionStorage` or `class session.MemoryStorage`.

I don't love `namespace`, it's just the only tool I know of that allows to accomplish this. Sadly, it leads to https://github.com/telegraf/telegraf/pull/1134#issuecomment-707634828.

Reply:
I agree completely that we shouldn't overwhelm new users, but I don't think that a large number of exports is too much of an issue here. I assume that most users either copy their code from the examples, or use their editor to automatically resolve them. I don't think many people will type `import {     } from 'telegraf'`, open autocomplete, and see what's available.